cdk/domino_cdk/config/iam.py

```diff
@@ -347,8 +347,6 @@ def do_cf():
             "eks:DescribeNodegroup",
             "eks:TagResource",
             "eks:UntagResource",
-            "ec2:RevokeSecurityGroupEgress",
-            "ec2:RevokeSecurityGroupIngress",
```

Comment:
This is a duplicate of these lines:
https://github.com/dominodatalab/cdk-cf-eks/blob/4d06375c75a5887c85bcdce51c95993262266e32/cdk/domino_cdk/config/iam.py#L396-L397

https://github.com/dominodatalab/cdk-cf-eks/blob/4d06375c75a5887c85bcdce51c95993262266e32/cdk/domino_cdk/config/iam.py#L413-L414

Reply:
Should we go opposite and remove * permissions?

Reply:
@Secretions preference?

Reply:
The only two things behind the wildcard are Authorize and Revoke, so the wildcard is probably fine.

Reply:
If there is only one additional operation that we intent to address with *, then definitely list it explicitly.

```
 "ec2:Authorize SecurityGroupEgress", 
 "ec2:Authorize SecurityGroupIngress", 
 "ec2:RevokeSecurityGroupEgress", 
 "ec2:RevokeSecurityGroupIngress", 
```